lib/sirsi_holding.rb

```diff
@@ -150,7 +150,7 @@ def to_item_display_hash
 
   # Call number normalization ported from solrmarc code
   def normalize_call_number(call_number)
-    return call_number unless %w[LC DEWEY].include?(call_number_type) # Normalization only applied to LC/Dewey
+    return call_number unless call_number && %w[LC DEWEY].include?(call_number_type) # Normalization only applied to LC/Dewey
```

Comment:
The call_number_type is set to LC or DEWEY.  That would be not be possible with a nil call_number.

Reply:
Again, am I missing something? The `call_number_type` method I'm looking at looks only at the scheme, and has nothing to do with the value of `call_number`. At least, in testing, I'm able to blow it up with:

`described_class.new(call_number: nil, scheme: 'LC', library: nil, home_location: nil, barcode: nil)`


```
  1) SirsiHolding#call_number with a nil call number returns an empty string
     Failure/Error: call_number = call_number.strip.gsub(/\s\s+/, ' ') # reduce multiple whitespace chars to a single space
     
     NoMethodError:
       undefined method `strip' for nil:NilClass
     # ./lib/sirsi_holding.rb:155:in `normalize_call_number'
     # ./lib/sirsi_holding.rb:44:in `call_number'
```

Reply:
I don't see why not? I don't think anything validates whether a call number is what you say it is.